__tests__/app.test.js

```diff
@@ -0,0 +1,44 @@
+const seed = require('../db/seeds/seed');
+const { categoryData, commentData, reviewData, userData } = require('../db/data/test-data');
```

Comment:
Here you destructure the properties of the data object, then input it as separate variables into the seed function, whereas you could just import and pass it to the seed function as one object, called something like 'testData' for example.